com.github.hopsan.Hopsan.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>se.hopsan.Hopsan</id>
```

Comment:
this isn't the same app-id. Failures will occur.

Reply:
fixed